rules/falco_rules.yaml

```diff
@@ -2708,7 +2708,17 @@
     (evt.arg.target in (sensitive_file_names) or evt.arg.target in (sensitive_directory_names))
   output: >
     Symlinks created over sensitive files (user=%user.name user_loginuid=%user.loginuid command=%proc.cmdline target=%evt.arg.target linkpath=%evt.arg.linkpath parent_process=%proc.pname)
-  priority: NOTICE
+  priority: WARNING
+  tags: [file, mitre_exfiltration]
+
+- rule: Create Hardlink Over Sensitive Files
```

Comment:
I believe we did notice quite a lot of false positives, that's why we keep the priority `NOTICE`

Reply:
Thats the main idea behind "Bypass rules via symlink creation" section of the writeup - its too easy to use a symlink bypass for WARNING rules, whereas the symlink creation by itself is only a NOTICE. Priority consistency from the attack detection perspective.

Reply:
It would be hard to iterate all the scenarios like secret, configmap etc. and `/var` directory is supposed to mean variant when Unix/Linux introduce it.

Reply:
okay, I am sort of okay with this one.